en/docs/connectors/catalog/communication/microsoft.teams/setup-guide.md

```
@@ -0,0 +1,84 @@
+
+   ![Add a client secret](/img/connectors/catalog/communication/microsoft.teams/setup/setup-guide-8.jpeg)
+
+2. Copy the secret **Value** immediately — it is shown only once. The **Secret ID** shown next to it is just a label for the secret, not a credential.
+
+   ![Certificates & secrets](/img/connectors/catalog/communication/microsoft.teams/setup/setup-guide-7.jpeg)
+
+## Obtain OAuth 2.0 credentials
+
+Microsoft Entra's v2.0 token endpoint for the registered application is:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)